src/targets/services/konnectorAlerts.js

```diff
@@ -35,17 +35,15 @@ const main = async ({ client }) => {
     ? (await client.query(Q(JOBS_DOCTYPE).getById(jobId))).data
     : undefined
 
-  // Used to execute a script on maif instance
-  // that force the execution of this service
-  // TODO should be removed after executing the script
-  if (serviceJob?.message?.forceIgnoredErrors) {
-    flag(
-      'banks.konnector-alerts.ignored-errors',
-      serviceJob.message.forceIgnoredErrors
-    )
+  const forcedIgnoredErrors =
+    serviceTrigger?.message?.forceIgnoredErrors ||
+    serviceJob?.message?.forceIgnoredErrors
+
+  if (forcedIgnoredErrors) {
```

Comment:
Should we keep the TODO to remove? 

Also I don't understand the change: it means that we store a "forceIgnoredErrors" in a trigger? Where do we do that?

Reply:
Non plus besoin du TODO. On n'utilise plus cette approche uniquement pour le script qu'on va exécuter sur le parc Maif, mais on l'utilise maintenant également de le code à la création du trigger https://github.com/cozy/cozy-banks/blob/feat/notifications/src/targets/services/konnectorAlerts/createTriggerAt.js#L21